src/pages/Login/Login.js

```diff
@@ -1,7 +1,89 @@
-import React from 'react';
+import { useState } from 'react';
+import UserInput from '../../components/UserInput/UserInput';
+import './Login.scss';
 
 const Login = () => {
-  return <div></div>;
+  const [loginInput, setLoginInput] = useState({
+    email: '',
+    password: '',
+  });
+  const loginInputHandler = e => {
+    const { name, value } = e.target;
+    setLoginInput({ ...loginInput, [name]: value });
+  };
+  const { email, password } = loginInput;
+
+  const emailRegex = /^[A-Za-z0-9_.-]+@[A-Za-z0-9-]+\.[A-Za-z0-9-]+/;
+  const emailCondition = emailRegex.test(email);
+  const pwdCondition = password.length > 4;
+
+  const loginFetch = () => {
+    fetch('/', {
+      method: 'POST',
+      body: JSON.stringify({
+        email,
+        password,
+      }),
+    })
+      .then(res => {
+        res.json();
+      })
+      .then(result => {
+        console.log('결과:', result);
+      });
+  };
+
+  return (
+    <div className="login">
+      <div className="container">
+        <div className="section-left">
+          <div className="login-header">
+            <span className="left-section-header">로그인</span>
+            <span>
+              신속한 결제 진행을 위해 필요한 정보를 요청할 수 있습니다.
+            </span>
+            <span>
+              Can’t find your order? You might have purchased as a guest. Click
+              and we’ll tell you how to find it.
+            </span>
+          </div>
+          <form className="login-main" onSubmit={loginFetch()}>
```

Comment:
```suggestion
          <form className="login-main" onSubmit={loginFetch}>
```
강철님! 해당 코드에서 ()가 있고 없고를 구현한 화면의 console 창을 통해
어떤 차이가 있는지 한번 확인해 보시겠어요?

Reply:
아아..ㅠ  () 을 넣어서 서브밋이 되기도 전에 실행이 되었습니다. 페이지가 로드 되고 바로 404 에러가 떴었습니다. +추가로 ()를 제거한 이후에 submit을 했더니 작동이 되지 않아서 e.preventDefault();를 fetch 함수 안에 추가 하였습니다. fetch 함수 자체에 e.preventDefault()를 넣는게 좋아보이지 않아서 아래와 같이 구현해 보았는데 괜찮을까요 !? 
onSubmit={e => {
              e.preventDefault();
              loginFetch();
            }}

Reply:
이벤트 속성을 사용할 때, 인라인으로 작성하고 싶다면 정말 한줄의 코드를 작성하거나 정말 부득이 한 경우에만 작성하는 것이 좋습니다
가독성 부분을 보더라도 e.preventDefault가 loginFetch 함수에 들어가는게 더 좋겠죠!

Reply:
그렇군요.. ! 확인 감사 드립니다. 수정해서 푸쉬 다시 올리도록 하겠습니다 ~